autoscheduler/frontend/src/components/LandingPage/HelpText.tsx

```diff
@@ -21,25 +21,25 @@ const HelpText: React.SFC = function App() {
             Guidelines
           </Box>
         </Typography>
-        <Typography
-          variant="body1"
-          align="center"
-        >
-          <Box padding={2}>
```

Comment:
Typography is rendered as a `<p>` component, so it can't have a `<div>` inside it